tests/object_utils_test.cpp

```diff
@@ -474,12 +24,11 @@ TEST(SphereWallObjectTest, RayInsideSphereHit) {
   RayHitData ignore;
 
   Vec3 dir(1, 2, 3);
-  Ray rayInsideSphere(sphere.getOrigin(), dir);
-  ASSERT_TRUE(sphere.hitObject(rayInsideSphere, kSkipFreq, &ignore))
-      << "Ray Hit didn't occur.";
+  Ray rayInsideSphere(sphere.getOrigin(), dir, kDefaultEnergyPerRay);
+  ASSERT_TRUE(sphere.hitObject(rayInsideSphere, kSkipFreq, &ignore));
 
-  Ray rayOutsideSphere(sphere.getOrigin() + Vec3(2 * kR, 0, 0), dir);
-  ASSERT_FALSE(sphere.hitObject(rayOutsideSphere, kSkipFreq, &ignore))
-      << "Ray hit when it shouldn't";
+  Ray rayOutsideSphere(sphere.getOrigin() + Vec3(2 * kR, 0, 0), dir,
+                       kDefaultEnergyPerRay);
```

Comment:
Having constants outside, makes this test really hard to understand, why hitObject should return false...

One would have to lookup what kDefaultBlah is and then go back here.